src/lib/layer.js

```diff
@@ -47,6 +47,10 @@ const defaultProps = {
   onDragMove: noop,
   onDragEnd: noop,
   onDragCancel: noop,
+  // Offset depth based on layer index to avoid z-fighting.
+  // Negative values pull layer towards the camera
+  // https://www.opengl.org/archives/resources/faq/technical/polygonoffset.htm
+  getPolygonOffset: ({layerIndex}) => [-layerIndex, -1],
```

Comment:
According to [https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glPolygonOffset.xml], we have

`
void glPolygonOffset(GLfloat factor, GLfloat units);
`
The value of the offset is factor × DZ + r × units , where DZ is a measurement of the change in depth relative to the screen area of the polygon, and r is the smallest value that is guaranteed to produce a resolvable offset for a given implementation. 

Shouldn't the params be [-1, -layerIndex]?